core/domain/skill_domain.py

```diff
@@ -412,25 +454,38 @@ def from_dict(cls, worked_example_dict):
             WorkedExample. The corresponding WorkedExample domain object.
         """
         worked_example = cls(
-            state_domain.SubtitledHtml(
+            state_domain.SubtitledHtml( # type: ignore[no-untyped-call]
                 worked_example_dict['question']['content_id'],
                 worked_example_dict['question']['html']),
-            state_domain.SubtitledHtml(
+            state_domain.SubtitledHtml( # type: ignore[no-untyped-call]
                 worked_example_dict['explanation']['content_id'],
                 worked_example_dict['explanation']['html'])
         )
-        worked_example.question.validate()
-        worked_example.explanation.validate()
+        worked_example.question.validate() # type: ignore[no-untyped-call]
+        worked_example.explanation.validate() # type: ignore[no-untyped-call]
 
         return worked_example
 
 
+class SkillContentsDict(TypedDict):
+    """Dictionary representing the Skill Contents Domain Object"""
+
+    explanation: Dict[str, state_domain.SubtitledHtml]
+    worked_examples: List[WorkedExampleDict]
+    recorded_voiceovers: Dict[str, Dict[str, state_domain.RecordedVoiceovers]]
+    written_translations: Dict[str, Dict[str, state_domain.WrittenTranslations]]
```

Comment:
For the dicts, aren't the values also a dictionary and not the domain objects? (So, the dict form of SubtitledHtml instead of a SubtitledHtml object?)

Reply:
@aks681 That'd mean creating separate `TypedDicts` for the `state_domain` domain objects, which'd be done once type annotations are added to `state_domain` or is there any alternative fix for this?

Reply:
Hmmm, @aks681 is right don't use `RecordedVoiceovers`, you will either need to wait for `state_domain` to be typed or add the `TypedDicts` yourself.